src/components/textInput/textInput.tsx

```diff
@@ -67,14 +69,22 @@ export default function TextInput(props: TextInputProps) {
         className="rustic-text-input"
         variant="outlined"
         value={messageText}
-        label={props.label}
+        placeholder={props.label}
```

Comment:
`label` and `placeholder` are two different things. They should not be set interchangeably.